src/views/v2/components/events-bar/views.php

```diff
@@ -23,6 +23,7 @@
 	'tribe-events-c-view-selector--labels' => empty( $disable_event_search ),
 	'tribe-events-c-view-selector--tabs'   => $is_tabs_style,
 ];
+$view_slug             = 'reflector' === $view_slug ? 'list' : $view_slug; // Test uses fake 'reflector' view.
```

Comment:
was this for a test? or are you actually using this?

Reply:
@paulmskim the test for it would fail cause it uses 'reflector' as view_slug instead of list, month, or whatever. I don't think there's a way to check if the template exists before rendering, right?

Reply:
I'm not too sure. Ask Luca T, he would know better. But we should fix this on the tests level, not on the template level.

Reply:
got the right solution after talking to Luca T, I'll update, thanks